view/theme/vier/dark.css

```diff
@@ -8,6 +8,9 @@ hr { background-color: #343434 !important; }
 a, .wall-item-name, .fakelink { 
 	color: #989898 !important; 
 }
+.wall-item-content a {
+	        font-weight: bold;
```

Comment:
I think changing it would be better then fetching up a PR that is 2 month old. ;-)